test/ocl_to_spirv_for_opencl_cpp.ll

```diff
@@ -0,0 +1,24 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
Consider adding a comment describing purpose of this test.   I think that looking at the filename or file history is not as clear or convenient.